### ISSUE TITLE
Can't check a URL from the command line

### ISSUE THREAD

**Issue by jaimeiniesta** [validator/validator#455]
I'm unable to check an URL from the command line like this:

```
$ java -version
java version "1.8.0_40"
Java(TM) SE Runtime Environment (build 1.8.0_40-b25)
Java HotSpot(TM) 64-Bit Server VM (build 25.40-b25, mixed mode)

$ java -jar ./vnu.jar --version
17.1.0

$ java -jar ./vnu.jar --format json https://github.com
Exception in thread "main" java.lang.NullPointerException
	at nu.validator.xml.PrudentHttpEntityResolver.resolveEntity(PrudentHttpEntityResolver.java:261)
	at nu.validator.validation.SimpleDocumentValidator.checkHttpURL(SimpleDocumentValidator.java:429)
	at nu.validator.client.SimpleCommandLineValidator.checkFiles(SimpleCommandLineValidator.java:243)
	at nu.validator.client.SimpleCommandLineValidator.main(SimpleCommandLineValidator.java:198)
```

But I'm able to get the page with `wget` and then validate the file:

```
$ wget https://github.com
$ java -jar ./vnu.jar --format json index.html
{"messages":[{"type":"error","url"...
```

**Comment by jaimeiniesta** [comment]
This works fine in the 16.6.29 release, but fails on 17.0.1 and 17.1.0

**Comment by sideshowbarker** [comment]
Thanks for catching this. I’ve released 17.2.0 with the fix https://github.com/validator/validator/releases/tag/17.2.0
